dockertest/subtestbase.py

```diff
@@ -168,6 +172,23 @@ def failif_not_in(needle, haystack, description=None):
                              (expect_s, description, haystack))
 
     @classmethod
+    def failif_not_redhat(cls, xcept=DockerTestNAError):
+        """
+        Raises an exception if this is a non-RHEL/RHELAH System
+
+        :param xcept: Non-default exception to raise
+        :raise DockerTestNAError: By default
+        """
+        try:
+            with open(cls.redhat_release_filepath, 'rb') as rhrel:
+                if 'Enterprise' not in rhrel.read():
+                    raise xcept("Expecting a Red Hat Enterprise Linux System")
```

Comment:
So I'm still a little hazy on when/how the `redhat/*` tests are invoked. (I'm actually a little hazy on whether docker-autotest is used anywhere outside RH). Are redhat/* tests invoked on all runs? On some? Is it configurable somehow?

The reason it matters is that, IMO, if run on all docker-autotest invocations, a more suitable exception message might be "this is not a RHEL system, move along" or equivalent. As stated above, "Expecting" implies that this is an unexpected situation.

PS Oooooh, nice! github: "instead of commenting right away, your comments will be grouped"!